Vuzix/app/src/main/java/com/facilitation/view/activities/spotify/SpotifyListActivity.kt

```diff
@@ -115,11 +116,10 @@ class SpotifyListActivity : AppCompatActivity(), ITapInput {
         recyclerView.adapter?.notifyDataSetChanged()
     }
 
-    private fun showSpotifySongActivity(selectedTrack: TrackDTO) {
+    private fun showSpotifySongActivity() {
         //TODO: Pass Spotify state to SongActivity - Jody 08.11.23
```

Comment:
Can this TODO be removed? As far as I can see, the state has been transferred to SongActivity in playSongFromList

Reply:
Yeah I think so